lib/discord.go

```diff
@@ -213,6 +265,48 @@ func GetBotUser(token string) (*BotUserResponse, error) {
 }
 
 func doDiscordReq(ctx context.Context, path string, method string, body io.ReadCloser, header http.Header, query string) (*http.Response, error) {
+	identifier := ctx.Value("identifier")
+	if identifier == nil {
+		// Queues always have an identifier, if there's none in the context, we called the method from outside a queue
+		identifier = "Internal"
+	}
+
+	logger.Info(method, path+"?"+query)
+
+	identifierStr, ok := identifier.(string)
+
+	if ok {
```

Comment:
We should probably not use the cache if the identifier is "Internal", I'm not sure if this is a big deal or not, but when "Internal" is there it means the queue is just being created, so checking the cache is meaningless anyway.